README.md

```diff
@@ -32,6 +32,8 @@ General guidelines for Hellbender developers
 
 * Untested code is considered **non-existent** and thus is subject to removal at any time (exception is main methods, or super corner conditions, or `toString()` code).
 
+* Each tool should have at least one good end-to-end integration test with a check for expected output, plus high-quality unit tests for all non-trivial utility methods/classes used by the tool. Although we have no specific coverage target, coverage should be extensive enough that if tests pass, the tool is guaranteed to be in a usable state.
```

Comment:
Guarantee is a strong word.  Perhaps we want that the author certifies it is in a usable state for its intended use?

Who defines the use cases for the integration or regression tests?  The tool developer?  The user of the tool?